LuaMenu/widgets/chobby/components/chat_windows.lua

```diff
@@ -818,17 +817,18 @@ function ChatWindows:GetPrivateChatConsole(userName, switchTo)
 	local privateChatConsole = self.privateChatConsoles[chanName]
 
 	if privateChatConsole == nil then
-		local function MessageListener(message)
-			if message:starts("/me ") then
-				lobby:SayPrivateEx(userName, message:sub(5))
-			else
-				lobby:SayPrivate(userName, message)
-			end
-		end
+
+		local sendPrivateMessageActions = {
```

Comment:
Console isn't just used for chat, it's used in a few more places that don't do special commands for "/me", e.g. 	self.debugConsole = Console(nil, MessageListener) . 
Also you missed some cases of Console being used (e.g. team and queue) so this would break things.

Prefer to just leave the MessageListener as it is

Reply:
I still suspect that it would be worth making the change – if you don't use `/me` then just don't give it an ircStyle argument or something – or maybe not. I feel like there's a better solution out there but whether there is or not, it should probably be handled in a different PR.

I'll revert these changes, and do some more thinking.